configs/deformable_detr/deformable_detr_r50_8x2_50e_coco.py

```diff
@@ -0,0 +1,171 @@
+_base_ = [
```

Comment:
8x2 seems to be a default setting, we can omit it for simplicity.

Reply:
should be 16*2.